tyr/security_groups/management.py

```diff
@@ -0,0 +1,35 @@
+rule = 'management'
+
+definition = {
+    'rules': [
+        {
+            'port': 22,
+            'source': [
+                'p-ops-vpn',
+                '@hudl-lincoln-east',
+                '@hudl-lincoln-west',
+                '@hudl-boston'
+                '@hudl-omaha'
+            ]
+        },
+        {
+            'port': 3389,
```

Comment:
one other though.. it might be nice to have constants for ports.. or at least comments on them stating what they're for.